core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt

```diff
@@ -91,9 +94,9 @@ class BattleTable(val worldScreen: WorldScreen): Table() {
         add("{Strength}: ".tr()+attacker.getAttackingStrength())
         add("{Strength}: ".tr()+defender.getDefendingStrength()).row()
 
-        val attackerModifiers = BattleDamage().getAttackModifiers(attacker,defender)  .map { it.key.tr()+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
+        val attackerModifiers = BattleDamage().getAttackModifiers(attacker,defender).map { (if(it.key.startsWith("vs ")) ("vs ["+it.key.replace("vs ","")+"]").tr() else it.key.tr())+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
```

Comment:
This is not readable as one line...

Reply:
i don't know how to solve it. it runs very well.
it means that:
for example, if it.key = "vs city" or "vs units in open terrain", we can use the method replace() to get the words "city" or "units in open terrain", at last we get the word "vs [city]" and "vs [units in open terrain]", i have translate the word "vs [unit]", so all words can be translated by tr().

Reply:
I have no problem with the logic, just with the readability.
Separate into separate lines

Reply:
now it's ok like you wish.